src/actions/sessionActions.js

```diff
@@ -12,14 +12,30 @@ export function resetSession(){
   return {type: types.RESET_SESSION};
 }
 
+export function configureSession(data){
+  return dispatch => {
+    targetClient.setUserInfo(data.token, data.user_id);
+    dispatch(updateSessionInformation("user_id", data.user_id));
+    dispatch(updateSessionInformation("user_token", data.token));
+    dispatch(updateSessionInformation("isLoggedIn", true));
+    cookie.save('user', data, { path: '/' });
```

Comment:
When you have time, try to migrate from the cookies to the local storage for save the session.